tools/misbehaving-jmx-server/README.md

```diff
@@ -33,3 +33,34 @@ $ docker run --rm -p :1099 misbehaving-jmx-server
 ```
 
 Can connect via jmxterm ` java -jar ~/jmxterm-1.0.2-uber.jar --url localhost:<rmi port>`
+
+## Testing the server with the Agent
+
+There are a couple of ways you can get the Agent to pull metrics from this test server.
+
+### JMX integration config
+
+Copy `misbehaving-jmxfetch-conf.yaml` to `/etc/datadog-agent/conf.d/` and just run the uber jar created by Maven.
+You will need to restart the Agent to pick up the config.
+
+### Using Docker
+
+After building the `misbehaving-jmx-server` you can simply run:
+
+```shell
+$ docker run \
+--rm \
+-p :1099 \
```

Comment:
I don't think this command works out of the box as docker binds container port 1099 to an ephemeral host port, but the config says to connect to port 1099.

```suggestion
-p 1099:1099 \
```